images/linux/scripts/installers/mysql.sh

```diff
@@ -9,19 +9,31 @@ source $HELPER_SCRIPTS/document.sh
 
 export ACCEPT_EULA=Y
 
-# Install MySQL Client
-apt-get install mysql-client -y
+if isUbuntu16 || isUbuntu18 ; then
+    apt-get install mysql-client -y
```

Comment:
not available on 20.04 yet?

Reply:
I have tried to install mysql 8 on clean ubuntu VM and mysql-client was installed as a part of mysql-server package.